packages/nextjs/src/runtime/node/fs.js

```diff
@@ -0,0 +1,12 @@
+const { existsSync, writeFileSync, readFileSync, appendFileSync, mkdirSync, rmSync } = require('node:fs');
+const path = require('node:path');
```

Comment:
i added `path` inside the #fs module because it was faster, let me know if you disagree